src/core/transaction/AssetTransferTransaction.ts

```diff
@@ -126,10 +174,19 @@ export class AssetTransferTransaction {
         });
     }
 
+    /**
+     * Get the outputs of this transaction.
```

Comment:
Are you sure that this is outputs instead of output?

Reply:
The method returns the array.